src/main/java/com/finki/pizzaslice/models/Order.java

```diff
@@ -1,14 +1,60 @@
 package com.finki.pizzaslice.models;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
+import java.util.Date;
 
 @Entity
-@Table(name = "orders")
+@Table(name = "orders",schema = "db")
 public class Order {
 
     @Id
-    private Integer id;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
 
+    @Column(name = "date_order")
+    private Date dateOrder;
+
+    @Column(name = "date_finish")
+    private Date dateFinish;
+
+    @Column
+    private Double price;
+
+    @Column
+    private Integer time;
+
+    @Column
+    private String promo;
+
+    @OneToOne
+    @JoinColumn(name = "fk_user",referencedColumnName = "id")
```

Comment:
Same as below comment in user, One order maps to One user, but one user can have multiple orders